ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/title/_cq_dialog/.content.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@rakeshkumar-verma : why have the chnages been made in the dialog . you could have modified teh design_dialog to add the p tag. refer screenshot below:

![image](https://user-images.githubusercontent.com/38690824/80367121-241d3a80-88a8-11ea-9205-d7666e47cf32.png)

Reply:
review comment incorporated

Reply:
review comment incorporated